Gemfile

```diff
@@ -99,12 +66,126 @@ gem "ruport",                         "=1.7.0",                       :git => "h
 # https://github.com/jeremyevans/ruby-american_date
 gem "american_date"
 
-group :ui_dependencies do # Added to Bundler.require in config/application.rb
+# Make sure to tag your new bundler group with the manageiq_default group in addition to your specific bundler group name.
+# This default is used to automatically require all of our gems in processes that don't specify which bundler groups they want.
+#
+### providers
+group :amazon, :manageiq_default do
+  manageiq_plugin "manageiq-providers-amazon"
+end
+
+group :ansible, :manageiq_default do
```

Comment:
Minor, but does this group mean ansible_tower, embedded_ansible or both?  If the latter, why not just have two group names.  Having it as `ansible` only feels confusing

```ruby
group, :ansible_tower, :embedded_ansible, :manageiq_default do
```